packages/components/package.json

```diff
@@ -40,7 +40,7 @@
 		"memize": "^1.0.5",
 		"moment": "^2.22.1",
 		"mousetrap": "^1.6.2",
-		"re-resizable": "^4.7.1",
+		"re-resizable": "^5.0.1",
```

Comment:
I'm curious on what caused this, but nice to see that we only needed a package bump here.

Reply:
I tried looking at the changelog, but I'm uncertain what caused it.

Reply:
renovate has a really nice package diff tool now that can help when the changelog is less than forthcoming: https://renovatebot.com/package-diff?name=re-resizable&from=4.7.1&to=5.0.1

Reply:
Though in this case... a lot changed :)

Reply:
nice I didn't know about that tool. 👍